core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -809,7 +809,7 @@ private void canonicalizeSchema(Schema schema) throws InvalidSchemaException {
       parsedSchema.rawSchema();
     } catch (Exception e) {
       log.error("Invalid schema " + schema);
-      throw new InvalidSchemaException("Invalid schema " + schema);
+      throw new InvalidSchemaException("Invalid schema " + schema, e);
```

Comment:
Can we build the string just once here ?

Reply:
Sure, good idea, thanks